src/v2/config.rs

```diff
@@ -112,15 +112,15 @@ impl Config {
                     // https://tools.ietf.org/html/rfc7231#section-5.3.2
                     (MediaTypes::ManifestV2S2, Some(0.5)),
                     (MediaTypes::ManifestV2S1Signed, Some(0.4)),
-                    // TODO(steveeJ): uncomment this when all the Manifest methods work for it
-                    // mediatypes::MediaTypes::ManifestList,
+                    (MediaTypes::ManifestList, Some(0.5)),
                 ],
                 // GCR incorrectly parses `q` parameters, so we use special Accept for it.
                 // Bug: https://issuetracker.google.com/issues/159827510.
                 // TODO: when bug is fixed, this workaround should be removed.
                 true => vec![
                     (MediaTypes::ManifestV2S2, None),
                     (MediaTypes::ManifestV2S1Signed, None),
+                    (MediaTypes::ManifestList, None),
```

Comment:
worth working in support for [`application/vnd.oci.image.index.v1+json`][1] at the same time, or different enough that we should defer that to later work?

[1]: https://github.com/opencontainers/image-spec/blob/v1.0.2/media-types.md#applicationvndociimageindexv1json

Reply:
We currently don't support OCI media types in dkregistry. https://github.com/camallo/dkregistry-rs/blob/master/src/mediatypes.rs

IMO we should be adding the support for OCI media types separately, as we'll be only supporting a single OCI media type if we end up adding [application/vnd.oci.image.index.v1+json](https://github.com/opencontainers/image-spec/blob/v1.0.2/media-types.md#applicationvndociimageindexv1json) to this PR